Tech@HubAPI/Tech@HubAPI/Models/Issue.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Tech_HubAPI.Models
+{
+    public class Issue
+    {
+        public Issue(string id, string title, User author, string activity, string buildInfo, string summary,string stepsToReproduce,
```

Comment:
Looks like you'll need to remove the `List<Comment> comments, List<User> assignees, List<Label> labels` parameters from the constructor. That's why the tests are failing I believe. Those fields are populated by the database ORM, not us, so I think that's why its giving an error. You can just initialize them to empty lists in the constructor.